src/client/components/PostInfo/index.js

```diff
@@ -4,7 +4,7 @@ class PostInfo extends React.Component {
   render() {
     return (
       <div className="post-info">
-        <span className="likes">{this.props.postInfo.numOfLikes} likes</span>
+        <span className="likes">{this.props.numOfLikes} likes</span>
```

Comment:
try to do this, with a data-* attribute and the before pseudo class. (just for fun and practicing) btw it will make the rendering faster